### PR TITLE
Ma/update runit

### PR DESCRIPTION
**Pull request by markan**
### Description

Update to newest runit and enterprise-chef-common cookbooks. Necessary to be able to move to Chef 15. Still a bit broken, work in progress.

Built on top of ma/remove_drbd, because the new enterprise_chef_common cookbook removes some drbd/keepalived logic

### Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussion that's relevant]

### Check List

- [ ] New functionality includes tests
- [x] All tests pass
- [x] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

File: omnibus/files/private-chef-cookbooks/private-chef/libraries/private_chef.rb [hunk 1]
```diff
@@ -290,7 +290,7 @@ def generate_hash
       results["private_chef"]["logs"]["log_retention"] = PrivateChef["log_retention"]
       results["private_chef"]["logs"]["log_rotation"] = PrivateChef["log_rotation"]
       results["private_chef"]["dark_launch"] = PrivateChef["dark_launch"]
-      results["private_chef"]["opscode-erchef"]["max_request_size"] = PrivateChef["opscode_erchef"]["max_request_size"]
+#      results["private_chef"]["opscode-erchef"]["max_request_size"] = PrivateChef["opscode_erchef"]["max_request_size"]
```
Comment (srenatus) [inline omnibus/files/private-chef-cookbooks/private-chef/libraries/private_chef.rb]: Intentional?

File: omnibus/files/private-chef-cookbooks/private-chef/recipes/postgresql.rb [hunk 2]
```diff
@@ -65,14 +65,16 @@
...
-  notifies :restart, 'runit_service[postgresql]' if is_data_master?
+  notifies :restart, 'component_runit_service[postgresql]', :delayed if is_data_master?
 end
 
 link postgresql_data_dir_symlink do
   to postgresql_data_dir
   not_if { postgresql_data_dir == postgresql_data_dir_symlink }
 end
 
+File.open("/dotfiles/node.json", "w") do |f| f.write node.to_json end
```
Comment (tyler-ball) [inline omnibus/files/private-chef-cookbooks/private-chef/recipes/postgresql.rb]: What does this do?

Comment (markan) [reply omnibus/files/private-chef-cookbooks/private-chef/recipes/postgresql.rb]: Debugging code; I was trying to figure out some details of what was going on internally

**Comment by markan** [comment]
Running in CI as https://buildkite.com/chef/chef-chef-server-master-omnibus-adhoc/builds/190

**Pull request merged**